### ISSUE TITLE
jsbeautify does not take 'javascript' as a type

### ISSUE THREAD

**Issue by bjagg** [google/vim-codefmt#95]
FormatCode fails on JavaScript files.
I am using version 1.7.5 of js-beautify. It takes 'js' instead of 'javascript' for type.

**Comment by malcolmr** [comment]
Right, it looks like the js-beautify CLI has always accepted --type={js,css,html}, but the Vim filetypes are javascript,json,css,html.

Both javascript and json should result in `--type=js`.
